docs/graphql/manual/guides/integrations/graphcool-migration.rst

```diff
@@ -0,0 +1,115 @@
+.. meta::
+   :description: Instructions to migrate your graph.cool project to Hasura
+   :keywords: hasura, docs, guide, GraphQL subscriptions, apollo, apollo-client
+
+.. _graphcool_migration:
+
+Migrate from graph.cool to Hasura
+=================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+If you are looking to migrate your graph.cool project to Hasura, these are the steps that you will need to roughly follow.
+
+Migrating data
+--------------
+
+Step 1: Export data
+^^^^^^^^^^^^^^^^^^^
+
+Export your Graph.cool data using their `export tool <https://export.graph.cool>`__. This will give a MySQL binary dump of your current data.
+
+.. thumbnail:: ../../../../img/graphql/manual/guides/graphcool-export.png
+   :alt: Graph.cool Export
+
+Step 2: Start MySQL with Docker
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+   docker run --name graphcool-mysql -e MYSQL_ROOT_PASSWORD=my-secret-pw -p 3306:3306 -d mysql:latest --default-authentication-plugin=mysql_native_password
```

Comment:
Do we have to mention here about dependencies that need to be installed for this command?